enterprise_subsidy/apps/subsidy/api.py

```diff
@@ -43,11 +43,14 @@ def get_or_create_learner_credit_subsidy(
     )
 
     if not subsidy.ledger:
-        ledger = create_ledger(unit=default_unit, idempotency_key=create_idempotency_key_for_subsidy(subsidy))
-        ledger.save()
+        ledger = create_ledger(
+            unit=subsidy.unit,
+            subsidy_uuid=subsidy.uuid,
+            inital_deposit=subsidy.starting_balance,
+        )
```

Comment:
This is a code smell to me: we're doing a very similar/identical thing in the `save_model()` method in the Django admin class for the subsidy model.  I think I might move this into a pre-save hook.